integration_tests/Containerfile

```diff
@@ -0,0 +1,82 @@
+# Use the base Debian image from the Podman hub
+FROM docker.io/library/debian
+
+
+
+# INSTALLING DEPENDENCIES
+
+# Update package repositories and install necessary packages
+RUN apt update -y
+
+# Copy and install dependencies before setup to save build time 
+COPY ../documentation/dependencies/debian.dependencies /dependencies/debian.dependencies
+COPY ./integration_tests/dependencies/kernel.dependencies /dependencies/kernel.dependencies
+
+RUN DEBIAN_DEPENDENCIES=$(cat /dependencies/debian.dependencies) \
```

Comment:
Check this: https://kworkflow.org/content/codingstyle.html#command-substitution-and-arithmetic-expression